[MAC] Electron rebuild issue on version v3.0.0

* Electron Version: `v3.0.0`
* Operating System (Platform and Version): `Mac`
* Last known working Electron version: `2.0.7`

**Expected Behavior**
Run `npm rebuild` should not fail

**Actual behavior**
```
$ npm rebuild

> electron@3.0.0-beta.5 postinstall /Users/${user}/Desktop/ele/node_modules/electron
> node install.js

/Users/${user}/Desktop/ele/node_modules/electron/install.js:47
  throw err
  ^

Error: EEXIST: file already exists, symlink 'Versions/Current/Electron Framework' -> '/Users/${user}/Desktop/ele/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Electron Framework'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! electron@3.0.0-beta.5 postinstall: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the electron@3.0.0-beta.5 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/${user}/.npm/_logs/2018-08-16T11_03_51_217Z-debug.log
```
**To Reproduce**
```
$ npm install electron@3.0.0-beta.5
$ npm rebuild
```

--- a/npm/install.js
+++ b/npm/install.js
@@ -17,7 +17,9 @@ try {
 
 var platformPath = getPlatformPath()
 
-if (installedVersion === version && fs.existsSync(path.join(__dirname, 'dist', platformPath))) {
+var electronPath = process.env.ELECTRON_OVERRIDE_DIST_PATH || path.join(__dirname, 'dist', platformPath)
+
+if (installedVersion === version && fs.existsSync(electronPath)) {
   process.exit(0)
 }
 

--- a/npm/install.js
+++ b/npm/install.js
@@ -17,7 +17,7 @@ try {
 
 var platformPath = getPlatformPath()
 
-if (installedVersion === version && fs.existsSync(path.join(__dirname, platformPath))) {
+if (installedVersion === version && fs.existsSync(path.join(__dirname, 'dist', platformPath))) {
   process.exit(0)
 }
 

